alicloud/resource_alicloud_polardb_account.go

```diff
@@ -0,0 +1,206 @@
+package alicloud
+
+import (
+	"fmt"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/polardb"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func resourceAlicloudPolarDBAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAlicloudPolarDBAccountCreate,
+		Read:   resourceAlicloudPolarDBAccountRead,
+		Update: resourceAlicloudPolarDBAccountUpdate,
+		Delete: resourceAlicloudPolarDBAccountDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"db_cluster_id": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"account_name": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"account_password": {
```

Comment:
please support kms_password to set a encrypted password. You can refer to alicloud_db_account or alicloud_instance